app/scripts/views/map/map-partial.html

```diff
@@ -18,6 +18,7 @@
                     typeahead-loading="loadingSuggestions"
                     typeahead-wait-ms="400"
                     typeahead-min-length="3"
+                    typeahead-on-select="searchMap()"
                     ng-submit="searchMap()"
```

Comment:
Are the similar handlers for `ng-submit` and `ng-keyup` still necessary?

Reply:
The `ng-submit` in particular I'm guessing never fires now, but I thought it wouldn't hurt to leave them in, in case it does work for some browser, or browser or plugin behavior should change again.

Going to do some cross-browser/OS testing with #271 that may be related.